config/testgrids/config.yaml

```diff
@@ -204,6 +204,15 @@ dashboards:
         value: 'ci-ovf-import-e2e-tests-daily issue: <test-name>'
       - key: body
         value: <test-url>
+  - name: ci-windows-upgrade-tests
```

Comment:
Please add this in the `/testgrids/conformance/conformance-all.yaml` file, with the dashboard that uses it.

Reply:
I think it has been added there. Did you review the latest commit?